plugin/kubernetes/controller.go

```diff
@@ -79,14 +87,21 @@ type dnsControlOpts struct {
 	// Label handling.
 	labelSelector *meta.LabelSelector
 	selector      labels.Selector
+	// watch-related items channel
```

Comment:
annoying that these are basically duplicated in Kubernetes and opts, but didn't want to refactor everything.